dot_env_example

```diff
@@ -2,89 +2,7 @@ HOSTNAME="localhost"
 PROJECT_CONTAINER_NAME="prefect"
 # This python path is added to avoid having to do relative imports.
 PYTHONPATH="/scripts/"
-DATAVERSE_URL="https://portal.staging.odissei.nl"
-DATAVERSE_API_TOKEN="example1-ab12-12ab-abcd-a1b2c3d4e5g6"
-XML2JSON_API_TOKEN="Bearer @km1-10122004-lamA"
 GITHUB_USERNAME="odissei-data"
 DOCKERHUB_USERNAME="fjodorvr"
 VERSION="v0.2.0-alpha"
```

Comment:
Zou het mogelijk zijn om deze laatste overblijfselen ook naar de dynaconf settings te krijgen? Lijkt me nice als we de heel .env kunnen verwijderen. Waarschijnlijk is de enige issue `ENV_FOR_DYNACONF=development`, maar misschien kunnen we dat als parameter van de docker compose command afhandelen.

Reply:
Ik heb zo min mogelijk aan de .env gezeten omdat ik niet zeker weet of Docker dan ontploft. Als we alle settings kunnen verhuizen doe ik dat graag.